plugins/Diagnostics/lang/en.json

```diff
@@ -0,0 +1,8 @@
+{
+    "Diagnostics": {
+        "ConfigFileTitle": "Config file",
+        "ConfigFileIntroduction": "Here you can view the Piwik configuration. If you are running Piwik in a load balanced environment the page might be different depending from which server this page is loaded. Rows with a different background color are changed config values that are specified for example in the %s file.",
+        "HideUnchanged": "If you want to see only changed values you can %shide all unchanged values%s.",
```

Comment:
As mentioned before. It's not easily possible to change the existing translations.
Guess it would take a few hours to update the english and afterwards all existing translations on transifex. Otherwise we would loose the existing translations. But maybe I'll try to do that step by step or plugin by plugin...